test/suites/para/test_tanssi_containers.ts

```diff
@@ -36,10 +36,12 @@ describeSuite({
             const relayNetwork = relayApi.consts.system.version.specName.toString();
             expect(relayNetwork, "Relay API incorrect").to.contain("rococo");
 
-            const paraNetwork = paraApi.consts.system.version.specName.toString();
             const paraId1000 = (await paraApi.query.parachainInfo.parachainId()).toString();
-            expect(paraNetwork, "Para API incorrect").to.contain("dancebox");
             expect(paraId1000, "Para API incorrect").to.be.equal("1000");
+            const paraNetwork = paraApi.consts.system.version.specName.toString();
```

Comment:
I dont think this suite is appropriate for flashbox, as it tests rotation which does not exist in flashbox. Maybe we should simply enable the suite without any test?

Reply:
Or am I wrong thinking that the suite tests rotation?

Reply:
Most importantly it tests registering and deregistering new chain. It also has a rotation happening in the middle of the test but that's mostly a coincidence. Since rotation is random there is a probability that the old assignment and the new assignment will be the same, so with that in mind this test should always pass in flashbox :)

Reply:
sounds good yeah! my main point was whether we test that the event thrown was `true` for rotation or something similar, but if not, then we are good to go